test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1167,6 +1167,27 @@ def forward(self, input1, input2, input3):
 
         self.run_test(TraceModel(), (x1, x2, x3), atol=10e-5)
 
+    def test_conv_str_padding(self):
+        class TraceModel(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = torch.nn.Conv1d(16, 33, 3, padding="valid")
+                self.conv2 = torch.nn.Conv2d(
+                    16, 33, (3, 5), stride=1, padding="valid", dilation=(3, 1)
+                )
+                self.conv3 = torch.nn.Conv3d(
+                    16, 33, (3, 5, 2), stride=1, padding="same"
```

Comment:
nit: Consider using parallel tests on multiple parameters combinations (ex: conv3d with padding="valid") to avoid missing tests.